### PR TITLE
eac: Add version 1.3

### PR DESCRIPTION
**Pull request by davidxuang**
A commonly-used audio CD grabber for accurate rip. [Website](http://www.exactaudiocopy.de/)

### CODE REVIEW

File: bucket/eac.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
+{
+    "version": "1.3",
+    "description": "A free audio CD grabber using standard drives.",
```
Comment (Ash258) [inline bucket/eac.json]:
```suggestion
    "description": "Exact Audio Copy using standard drives",
```

File: bucket/eac.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    "homepage": "http://www.exactaudiocopy.de/",
```
Comment (Ash258) [inline bucket/eac.json]:
```suggestion
    "homepage": "http://www.exactaudiocopy.de",
```

File: bucket/eac.json [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    "license": "Freeware",
+    "url": "http://www.exactaudiocopy.de/eac-1.3.exe#dl.7z",
```
Comment (Ash258) [inline bucket/eac.json]: ...

```suggestion
    "url": "http://www.exactaudiocopy.de/eac-1.3.exe#/dl.7z",
```

File: bucket/eac.json [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+    "hash": "sha1:49f1028bd2b0cce0829250430bf8771c4a766daf",
+    "post_install": "Remove-Item \"$dir\\`$PLUGINSDIR\", \"$dir\\Empty*\", \"$dir\\uninst*\" -Recurse",
```
Comment (Ash258) [inline bucket/eac.json]:
```suggestion
    "pre_install": "Remove-Item \"$dir\\`$PLUGINSDIR\", \"$dir\\Empty*\", \"$dir\\uninst*\" -Recurse",
```

File: bucket/eac.json [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+    "shortcuts": [
+        [
+            "EAC.exe",
+            "Exact Audio Copy"
+        ]
+    ],
+    "checkver": {
+        "url": "http://www.exactaudiocopy.de/en/index.php/resources/download/",
+        "regex": "V([\\d.]+)"
```
Comment (Ash258) [inline bucket/eac.json]: Too generic

```suggestion
        "regex": "Exact Audio Copy\\s+V([\\d.]+)"
```

File: bucket/eac.json [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+    "url": "http://www.exactaudiocopy.de/eac-1.3.exe#dl.7z",
+    "hash": "sha1:49f1028bd2b0cce0829250430bf8771c4a766daf",
+    "post_install": "Remove-Item \"$dir\\`$PLUGINSDIR\", \"$dir\\Empty*\", \"$dir\\uninst*\" -Recurse",
+    "shortcuts": [
+        [
+            "EAC.exe",
+            "Exact Audio Copy"
+        ]
+    ],
+    "checkver": {
```
Comment (Ash258) [inline bucket/eac.json]: Is it needed? 

> Exact Audio Copy V1.3 (including CDRDAO and Flac packages) last changed on **04.09.16**

File: bucket/eac.json [hunk 7]
```diff
@@ -0,0 +1,26 @@
...
+        "url": "http://www.exactaudiocopy.de/en/index.php/resources/download/",
+        "regex": "V([\\d.]+)"
+    },
+    "autoupdate": {
+        "url": "http://www.exactaudiocopy.de/eac-$version.exe#dl.7z",
```
Comment (Ash258) [inline bucket/eac.json]:
```suggestion
        "url": "http://www.exactaudiocopy.de/eac-$version.exe#/dl.7z",
```

File: bucket/eac.json (hunk 6 above)
Comment (davidxuang) [reply bucket/eac.json]: I suggest it may update in the future, as FreeDB, the CD metadata library it depends, is shutting down.

**Pull request merged**